components/provisioner/internal/model/types.go

```diff
@@ -24,18 +24,21 @@ const (
 type OperationStage string
 
 const (
-	ShootProvisioning      OperationStage = "ShootProvisioning"
-	StartingInstallation   OperationStage = "StartingInstallation"
-	WaitingForInstallation OperationStage = "WaitingForInstallation"
-	ConnectRuntimeAgent    OperationStage = "ConnectRuntimeAgent"
-	WaitForAgentToConnect  OperationStage = "WaitForAgentToConnect"
+	//ShootProvisioning               OperationStage = "ShootProvisioning"
```

Comment:
We need to bare in mind that there may be existing clusters with this stage so we need to handle them somehow, either by database migration or in code.

Reply:
Good catch. I will handle this in the code.

Reply:
Done.